packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -53,12 +53,15 @@ const chatMessageStyles: ComponentSlotStylesInput<
 
     ...getBorderFocusStyles({ siteVariables, isFromKeyboard: p.isFromKeyboard }),
 
-    ':hover': {
-      [`& .${ChatMessage.slotClassNames.actionMenu}`]: {
-        opacity: 1,
-        width: 'auto',
+    ...(v.showActionMenu === undefined && {
+      ':hover': {
```

Comment:
I think that it would be helpful to provide short comment on why this variable's value is checked, to clearly suggest its intent. Something like: 

> actions menu's appearance can be controlled by value of `showActionMenu` variable - in this case this variable will serve the single source of truth on whether actions menu should be shown. Otherwise, if variable is not provided, the default appearance logic will be used for actions menu.